README.md

```diff
@@ -189,19 +189,20 @@ curl -d "insert data { graph <http://example.org/> { <urn:a> <urn:b> <urn:c> } }
 ```
 
 ### Provenance Interface
+To use the provenance feature you have to enable it with the argument `--feature=provenance`.
```

Comment:
Could you please additionally add an info text whether this feature flag just enables the feature (sparql endpoint) or whether it enables the whole provenance functionality? So to answer: If provenance is disabled, is it not record any provenance data at all or is it just not providing a sparql endpoint to query the recorded data?